src/main/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveClient.kt

```diff
@@ -66,6 +66,7 @@ class OppgaveClient @Autowired constructor(@param:Value("\${FAMILIE_INTEGRASJONE
         }.fold(
                 onSuccess = { response -> assertGyldig(response) },
                 onFailure = {
+                    log.warn("Post-kall mot $uri feilet ved opprettelse av oppgave", it)
```

Comment:
Hvorfor trenger vi egen warn her? Vil ikke exception'en bli logget i apiExceptionHandler?

Reply:
La det mest til for e2e. Feilmeldingen havner som tekst på tasken som kjører kallet. Noe som gjør det knotete å finne ut hva som feiler. Særlig hvis man ikke kjører lokalt og har mulighet til å sjekke securelog på podden